nsgcli/show.py

```diff
@@ -202,7 +198,7 @@ def do_views(self, arg):
                         print(response.content)
                     else:
                         filtered = {}
-                        for k in response.keys():
+                        for k in list(response.keys()):
```

Comment:
In python3, type of dict.keys() is changed to `dict_keys` from `list`. This could lead to indexing error. So have wrapped it around list to get list of keys. 

After going through the code I see we are not doing any index operation, so we could remove list converting. I will make this change at relevant places.

Reply:
ok